analytics/migrations/0002_submissionanalytics_time_created.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.0.14 on 2022-01-24 03:12
```

Comment:
You should only need to add 1 migration. All the changes should be here.

Reply:
Done.